common/StreakHandler.ts

```diff
@@ -29,7 +29,7 @@ export const readAppStateAndCheckStreak = async (): Promise<AppState | null> =>
     state = await readAppState(); // read the state from storage
     if (!state) return null;
     if (!state.dailyStreak) state.dailyStreak = 0;
-    if (!state.lastLogin) state.lastLogin = Date.now();
+    if (!state.lastLogin) state.lastLogin = new Date().getTime(); // Current UTC time
```

Comment:
It's not UTC time, but a unix epoch timestamp (i.e., the number of millis since 1.1.1970)
new Date().getTime()
1609767500130